index-import.cpp

```diff
@@ -384,8 +384,10 @@ static bool remapIndex(const Remapper &remapper,
   SmallString<256> outputRecordsDirectory;
   path::append(outputRecordsDirectory, OutputIndexPath, "v5", "records");
 
+  bool recordsDirectoryExists = fs::exists(recordsDirectory);
+
   if (not fs::is_directory(unitDirectory) ||
-      not fs::is_directory(recordsDirectory)) {
+      (recordsDirectoryExists && not fs::is_directory(recordsDirectory))) {
```

Comment:
you can just remove this condition instead, i wouldn't worry about handling it being a non-directory